test/functional/page_objects/dashboard_page.js

```diff
@@ -525,8 +525,10 @@ export function DashboardPageProvider({ getService, getPageObjects }) {
     async filterOnPieSlice(sliceValue) {
       log.debug(`Filtering on a pie slice with optional value ${sliceValue}`);
       if (sliceValue) {
+        log.debug('Clicking slice');
```

Comment:
The line below should output debug value of something like `testSubjects.click(pieSlice-blue)`.  Was that not enough?

Reply:
Oops, I think this is leftover from a debugging session. I'll remove these logs, thanks for calling them out!